soc/arm/nuvoton_npcx/common/soc_dt.h

```diff
@@ -166,6 +157,114 @@
 		     inst)                         \
 	}
 
+/**
+ * @brief Node identifier for an instance of a specific compatible
+ *
+ * @param compat specific compatible of devices in device-tree file
+ * @param inst instance number
+ * @return a node identifier for the node with "io_comp" compatible and
+ *         instance number "inst"
+ */
+#define DT_NPCX_COMP_INST(compat, inst) DT_INST(inst, compat)
```

Comment:
Please don't use `DT_` prefix for such macros, we try and reserve the name space for system DT macros.

Reply:
I checked all macros in "soc_dt.h" and modified the macros which prefix doesn't meet DT_NPCX that follows ATMEL did here(https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/atmel/pinctrl_atmel_sam.h#L39). Is it OK with you?
Or is NPCX_DT prefix better?

Reply:
I think the atmel macros are intended for use in devicetree sources, as they generate pinmux data, so they should have a `DT_` prefix.

       soc {
                pinctrl@400e0e00 {
                        /* instance, signal, pio, pin, peripheral */
                        DT_ATMEL_PIN(spi0, miso, a, 25, a);
                        DT_ATMEL_PIN(spi0, mosi, a, 26, a);
                        ...

These macros instead are used in C code to extract information from translated devicetree sources.  They *probably* should be `NPCX_DT_` since they don't live in `include/devicetree/*` where other similar system-wide macros are placed.

But @galak can provide authoritative direction.

Reply:
Re @pabigot:
Thanks. it's clear to me. I will replace all macros for constructing with NPCX_DT_ prefix. to prevent indistinct usage.